src/pages/signup/Signup.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  IonContent,
+  IonHeader,
+  IonPage,
+  IonCard,
+  IonCardContent,
+  IonCardHeader,
+  IonCardTitle,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonInput,
+  IonButton,
+  IonIcon,
+  useIonAlert,
+} from "@ionic/react";
+import { logoGoogle } from "ionicons/icons";
+//import ExploreContainer from '../../components/ExploreContainer';
+import "./Signup.css";
+
+const Signup: React.FC = () => {
+  const [presentAlert] = useIonAlert();
+
+  function validateEmail() {
+    presentAlert({
+      header: "Alert",
+      cssClass: "center-alert",
+      subHeader: "There is already an existing account under this email",
```

Comment:
This might be a no-no from a security perspective. I was told never to do this when I did my OWASP security training but it seems like there's some discussion around it: https://security.stackexchange.com/questions/158075/is-it-unsafe-to-show-message-that-username-account-does-not-exist-at-login 
TLDR; A user should not be able to find out if an account exists or not without signing in. I don't know how to avoid this.

Reply:
Most websites just say "That username and password combination does not exist" or something similar, then rely solely on the "forgot password" link to let someone reset their password. It also doesn't indicate on the forgot password form whether an email exists or not, just says that an email has been sent.

Reply:
@abluescarab That's a good point but we're not talking about forgotten passwords. This is the scenario where someone already has an account and they try to make another one with the same email.